server/game/GameActions/SequentialMakeTokenCreatureAction.js

```diff
@@ -0,0 +1,111 @@
+const SingleCardSelector = require('../CardSelectors/SingleCardSelector');
+const GameAction = require('./GameAction');
+
+class CardListSelector extends SingleCardSelector {
```

Comment:
I believe we had a PR that turn this `CardListSelector` inner class into a file itself. I think we should reuse.

Reply:
Weird, I don't know how that change got lost.  Fixed!